src/sessionization.py

```diff
@@ -0,0 +1,36 @@
+# stdlib
+import csv
+import sys
+
+# first party
```

Comment:
would be nice to have brief comments here describing what this file does...also not sure what 'first party' means